flow/crc32c.cpp

```diff
@@ -171,7 +173,7 @@ static inline uint32_t shift_crc(uint32_t shift_table[][256], uint32_t crc)
 }
 
 /* Compute CRC-32C using the Intel hardware instruction. */
-#if defined(__clang__) || defined(__GNUG__)
+#if (defined(__clang__) || defined(__GNUG__)) && (!defined(__aarch64__))
```

Comment:
This change means that the comment above is now not correct anymore. Can you please update the comment as well?

Reply:
Ah, yeah, good catch.  Will fix the comment.